app-backend/db/src/main/scala/com/azavea/rf/database/Dao.scala

```diff
@@ -62,45 +62,49 @@ object Dao {
       this.copy(filters = filters ++ filterable.toFilters(Some(fr"id = ${id}")))
     }
 
-    def authorizeFragment(user: User, objectType: ObjectType, actionType: ActionType): Option[Fragment] = Some(
-      fr"""id IN (
-        -- Collect objects owned by the user
-        SELECT A.id
-        FROM""" ++ tableF ++ fr"""AS A
-        WHERE A.owner = ${user.id}
-
-        UNION ALL
-
-        -- Collect objects the user has access to for non-group permissions
-        SELECT A.id
-        FROM""" ++ tableF ++ fr"""AS A
-        JOIN access_control_rules acr ON
-          acr.object_id::text = A.id::text
-          WHERE
-            acr.object_type = ${objectType} AND
-            acr.action_type = ${actionType} AND
-            -- Match if the ACR is an ALL or per user
-            (
+    // Filter to validate access on an object type
+    def authorizeFragment[M >: Model](user: User, objectType: ObjectType, actionType: ActionType)(implicit filterable: Filterable[M, Option[Fragment]]): QueryBuilder[Model] = {
+      if (user.isSuperuser) {
+        Some(fr"true")
+      } else {
+        Some(
+          fr"""id IN (
+            -- Collect objects owned by the user
+            SELECT A.id
+            FROM""" ++ tableF ++ fr"""AS A
+            WHERE A.owner = ${user.id}
+
+            UNION ALL
+
+            -- Collect objects the user has access to for non-group permissions
+            SELECT A.id
+            FROM""" ++ tableF ++ fr"""AS A
```

Comment:
We updated the table join logic during an offline pairing yesterday. This seems to be back to what it was before. It might be a rebase related issue.